upi/vsphere/bootstrap/main.tf

```diff
@@ -23,12 +23,12 @@ data "vsphere_virtual_machine" "template" {
 }
 
 resource "vsphere_virtual_machine" "vm" {
-  name             = "bootstrap-${var.cluster_id}"
+  name             = "bootstrap1"
```

Comment:
nit: i guess there is no need for an int otherwise it gives the impression that `bootstrap{2,3}` could follow

Reply:
Good add. I removed the numbering scheme from bootstrap.